.github/actions/frontend-shared-version/index.js

```diff
@@ -0,0 +1,33 @@
+const core = require('@actions/core');
+const util = require('util');
+
+const exec = util.promisify(require('child_process').exec);
+
+/**
+ * Increment the minor part of a `MAJOR.MINOR.PATCH` semver version.
+ */
+function bumpMinorVersion(version) {
+  const parts = version.split('.')
+  if (parts.length !== 3) {
+    throw new Error(`${version} is not a valid MAJOR.MINOR.PATCH version`);
+  }
+  const newMinorVersion = parseInt(parts[1]) + 1;
+  return `${parts[0]}.${newMinorVersion}.${parts[2]}`
+}
```

Comment:
It might be possible to do this more succinctly and with additional validation using the `semver` package, but I think this is fine for the moment.

Reply:
I think we can do that, perhaps as a follow up?